uint64 waiting_time
string reminder_type
---
bool success